src/serac/physics/nonlinear_solid.cpp

```diff
@@ -314,19 +314,19 @@ void NonlinearSolid::InputOptions::defineInputFileSchema(axom::inlet::Table& tab
   table.addDouble("viscosity", "Viscosity constant").defaultValue(0.0);
 
   auto& stiffness_solver_table =
-      table.addTable("stiffness_solver", "Linear and Nonlinear stiffness Solver Parameters.");
+      table.addStruct("stiffness_solver", "Linear and Nonlinear stiffness Solver Parameters.");
   serac::mfem_ext::EquationSolver::DefineInputFileSchema(stiffness_solver_table);
 
-  auto& dynamics_table = table.addTable("dynamics", "Parameters for mass matrix inversion");
+  auto& dynamics_table = table.addStruct("dynamics", "Parameters for mass matrix inversion");
   dynamics_table.addString("timestepper", "Timestepper (ODE) method to use");
   dynamics_table.addString("enforcement_method", "Time-varying constraint enforcement method to use");
 
   auto& bc_table = table.addGenericDictionary("boundary_conds", "Table of boundary conditions");
```

Comment:
While we're changing `addTable` -> `addStruct`, we might also want to change `addGenericDictionary` -> `addStructDictionary` and `addGenericArray` -> `addStructArray`.